Projects/hCore/Sources/UIDevice+modelName.swift

```diff
@@ -95,7 +95,7 @@ extension UIDevice {
                 case "i386", "x86_64", "arm64":
                     return
                         "Simulator \(mapToDevice(identifier: ProcessInfo().environment["SIMULATOR_MODEL_IDENTIFIER"] ?? "iOS"))"
-                default: return identifier
+                    defaultStyle: return identifier
```

Comment:
This probably got automatically changed for you xD Xcode is not the best tool for renaming xD